### PR TITLE
UHM-7779: Make OCL the default/only option for concept loading

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/pihcore/config/PihConfigService.java [hunk 1]
```diff
@@ -94,20 +92,6 @@ public InitializerConfig getInitializerConfig() {
...
-        if (mdsFiles != null) {
-            for (File f : mdsFiles) {
-                ConfigFile mdsFile = new ConfigFile();
-                mdsFile.setPath(f.getName());
-                mdsFile.setEnabled(true);
-                pihConceptsDomain.getConfigFiles().add(mdsFile);
-            }
-        }
-        return pihConceptsDomain;
-    }
```
Comment (mogoodrich) [inline api/src/main/java/org/openmrs/module/pihcore/config/PihConfigService.java:109]: @mseaton  I wasn't 100% sure what this was used for, can we confirm it's not longer needed?

**Review by mseaton — APPROVED** [review]
Great work, I love this kind of cleanup :)

**Review by lnball — APPROVED** [review]
Very satisfying cleanup.  Does this mean we don't use mds module anymore?

**Comment by mogoodrich** [comment]
@lnball , yes, it should be going away from the distro as part of this change.

Thanks, merging!

**Pull request merged**